source/crud/tests/v2/default-write-concern.yml

```diff
@@ -0,0 +1,413 @@
+data:
+  - _id: 1
+    x: 11
+  - _id: 2
+    x: 22
+collection_name: default_write_concern_coll
+database_name: default_write_concern_db
+tests:
+  - description: Aggregate with $merge
+    operations:
+      - object: collection
+        name: aggregate
+        arguments:
+          pipeline: &ref_0
+            - $match:
+                _id:
+                  $gt: 1
+            - $merge:
+                into: other_test_collection
+    expectations:
+      - command_started_event:
+          command:
+            aggregate: default_write_concern_coll
+            pipeline: *ref_0
+            writeConcern: null
+    outcome:
+      collection:
+        name: other_test_collection
+        data:
+          - _id: 2
+            x: 22
+  - description: Aggregate with $out
+    operations:
+      - object: collection
+        name: aggregate
+        arguments:
+          pipeline: &ref_1
+            - $match:
+                _id:
+                  $gt: 1
+            - $out: other_test_collection
+          batchSize: 2
+    outcome:
+      collection:
+        name: other_test_collection
+        data:
+          - _id: 2
+            x: 22
+    expectations:
+      - command_started_event:
+          command:
+            aggregate: default_write_concern_coll
+            pipeline: *ref_1
+            writeConcern: null
+  - description: runCommand with a write command
+    operations:
+      - object: database
+        name: runCommand
+        arguments:
+          command:
+            create: test_collection
+    expectations:
+      - command_started_event:
+          command:
+            create: test_collection
+            writeConcern: null
+  - description: create and drop index
+    operations:
+      - object: collection
+        name: createIndex
+        arguments:
+          model:
+            x: 1
+      - object: collection
+        name: dropIndex
+        arguments:
+          name: _x_1
+    expectations:
+      - command_started_event:
+          command:
+            createIndexes: default_write_concern_coll
+            indexes:
+              - name: _x_1
+                key:
+                  x: 1
+            writeConcern: null
+      - command_started_event:
+          command:
+            dropIndexes: default_write_concern_coll
+            index: _x_1
+            writeConcern: null
+  - description: delete one
```

Comment:
Existing tests refer to this the operation as a capitalized, camel-case term (DeleteOne in this case). So I'd suggest using "DeleteOne omits default write concern" here. FWIW, the existing Aggregate tests are already consistent with this, but I think the index ones could be "CreateIndex and DropIndex omit default write concern".

Reply:
Done.